pyseir/parameters/parameter_ensemble_generator.py

```diff
@@ -39,10 +39,9 @@ def __init__(self, fips, N_samples, t_list, I_initial=1, suppression_policy=None
                 load_data.load_county_metadata().set_index("fips").loc[fips].to_dict()
             )
             self.state_abbr = us.states.lookup(self.county_metadata["state"]).abbr
-            self._latest = combined_datasets.get_us_latest_for_fips(self.fips)
         else:
             self.state_abbr = us.states.lookup(fips).abbr
```

Comment:
Are `self.state_abbr` and `self.county_metadata` used anywhere else? You may be able to remove this whole block

Reply:
Good call! done.